### PR TITLE
Add unit tests for existing project functionality

### PR DESCRIPTION
**Pull request by dwkletke**
This PR adds unit tests in state_estimation/tests for the existing code involved in data generation. In addition, there are a few minor tweaks to the Rocket class in rocket_math.py and overall structure of data_generation.py (renamed from SES.py). Please let me know if I am missing any obvious test cases!

Note: anything to do with sensor data/generation is to be looked at/implemented after this PR.

---
This change is [Reviewable](https://reviewable.io/reviews/waterloo-rocketry/state_estimation/1)

### CODE REVIEW

**Comment by dwkletke** [comment]
Oh also: I added a requirements.txt in case anyone needed to install the required packages.

File: tests/test_rocket_math.py [hunk 1]
```diff
@@ -0,0 +1,752 @@
...
+import rocket_math as rm
+
+'''
+Notes:
+- For all calculations, there will be an arbitrary 4 decimal places max.
+- No rocket_air_density unit test since the current method for
+determining/testing air density needs to be changed.
+'''
+
+# Blank Rocket object to be used to initialize rockets in functions.
```
Comment (usephysics) [inline tests/test_rocket_math.py:14]: big problem with this - you only create one Rocket object, and it's all passed by reference, so every function that you create "test_rocket" is still pointing to this blank_rocket. all modifications you make are global. assuming this isn't intended, you'll need to instantiate a new Rocket for each function

**Comment by usephysics** [comment]
code looks pretty good, only major thing is the last comment I had in the review (if this is unintended).

**Comment by usephysics** [comment]
> I'm on the fence about using imperial units for internal calculation. I know we use them for most of our CAD and design...but I do think that it might make things harder and more annoying to read internally.
> 
> > _[Reviewable](https://reviewable.io/reviews/waterloo-rocketry/state_estimation/1#-:-MFlth_X5Z_G93XzHl-U:b-po7if5)_ status: 1 of 7 files reviewed, 25 unresolved discussions (waiting on @dwkletke, @jacobdeery, @usephysics, and @wendi-yu)

I was thinking this too, I wasn't sure if it was because of CAD or because SAC uses feet, but that metric would probably be better in this context

**Comment by dwkletke** [comment]
The initial motivation for having imperial units was because we use that for most things (SF, CAD etc). If you guys are comfortable with the change from imperial to metric, I am happy to do so.

**Pull request merged**